contracts/DaoV2.sol

```diff
@@ -125,14 +130,15 @@ contract Dao {
         mStatus =false;
         secondsPerEra = iBASE(BASE).secondsPerEra();
     }
-    function setGenesisAddresses(address _router, address _utils, address _lend, address _bond, address _daoVault, address _poolFactory,address _synthFactory ) public onlyDAO {
+    function setGenesisAddresses(address _router, address _utils, address _lend, address _bond, address _daoVault, address _poolFactory,address _synthFactory, address _reserve ) public onlyDAO {
         _ROUTER = iROUTER(_router);
         _UTILS = iUTILS(_utils);
         _LEND = iLEND(_lend);
         _BOND = iBOND(_bond);
         _DAOVAULT = iDAOVAULT(_daoVault);
         _POOLFACTORY = iPOOLFACTORY(_poolFactory);
         _SYNTHFACTORY = iSYNTHFACTORY(_synthFactory);
+        _RESERVE = iRESERVE(_reserve);
```

Comment:
Add reserve genesis address ('Reserve' will handle SPARTA emissions instead of 'Router')